apps/team/src/pages/api/image/list.ts

```diff
@@ -1,6 +1,6 @@
-import { imageCache } from "lib/models";
+import { imageCache } from "@artsy-studio/models";
```

Comment:
What does this do? Is a "model" in the conventional sense, or is it a util?

Reply:
It stores team nav resized photo links in a model that's persisted to simpledb. So a model in the rails sense.

Reply:
If this is specific to team nav can it live there instead of at the framework level?

Reply:
Well, it's built in a way not to be specific to team nav. I'm honestly not sure yet. Persistence (and other ways of accessing data) is one of the big things I want to spend more time thinking about next. 

For now this is the only way to persist data and I'd like to leave it shareable until we have other mechanisms.

Reply:
Down to iterate but with a caveat: Since the idea is to scale things up so that each app can live at whatever scale, I feel like the _requirement_ that this currently team-nav-specific perisistence be in the global scope might be a pointer to a larger problem that we might want to address sooner rather than later so that apps dont end up winding around one another and bloating framework-level scope.

Reply:
> currently team-nav-specific perisistence

Nothing about this is specific to team nav. It's a metadata store specifically for referencing image URLs stored in S3 for a faster lookup. 

To alleviate the concern, I'll move it into team until it's needed elsewhere or we come up with an alternative storage mechanism.